packages/slopShell/PKGBUILD

```diff
@@ -0,0 +1,46 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=slopShell
+pkgver=1
+_SLOP_DIR="/opt/$pkgname-$pkgver/"
+pkgrel=1
+pkgdesc='PHP based backdoor/webshell with C2 capabilities'
+arch=('any')
+groups=('blackarch', 'blackarch-backdoor')
+url='https://github.com/oldkingcone/slopShell'
+license=('Apache License 2.0')
```

Comment:
It's just Apache

- https://wiki.archlinux.org/title/PKGBUILD#license
- https://archlinux.org/packages/core/any/licenses/